### PR TITLE
trigger metabase test run instead of including it as a step here

### PR DESCRIPTION
**Pull request by toshok**
https://linear.app/replay/issue/RUN-3072/make-metabase-e2e-tests-blocking

Trigger a workflow in the metabase repo that triggers the GH action with the proper inputs, then awaits completion of that workflow in a flyvm so we don't tie up any buildkite agents. phew.

depends on https://github.com/replayio-public/metabase/pull/108

### CODE REVIEW

File: replay_build_scripts/metabase.sh [hunk 1]
```diff
@@ -32,7 +34,7 @@ if [ -n "$2" ]; then
     FOLDERS="${FOLDERS%,}]\""
 fi;
 
-INPUTS='{"ref":"master","inputs":{"chromium-build-id":"'${BUILD_ID}'"'${FOLDERS}'}}'
+INPUTS='{"ref":"toshok/workflow-dispatcher-chooses-run-name","inputs":{"runId:"'${RUN_ID}'","chromium-build-id":"'${BUILD_ID}'"'${FOLDERS}'}}'
```
Comment (toshok) [inline replay_build_scripts/metabase.sh:37]: make sure to change `ref` back to master once the metabase PR lands

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -121,7 +121,6 @@ steps:
   - label: "Metabase Test Suite"
     key: "metabase-test-suite"
     depends_on: "build-chromium-linux-x86_64"
-    if: build.branch == "master"
```
Comment (toshok) [inline .buildkite/pipeline.yml:124]: did this just to get the buildkite step to run for this PR, but is there a reason we shouldn't be running the metabase tests always?

Comment (toshok) [reply .buildkite/pipeline.yml]: (current answer: they take a while to run.)

File: replay_build_scripts/metabase.sh (hunk 1 above)
Comment (toshok) [reply replay_build_scripts/metabase.sh]: and back to `master`

**Comment by Domiii** [comment]
You are already allowed to merge since non-src changes are auto-approved.
@jazzdan or @klochek might be more helpful in an "assist review" than I am with GH actions and friends 👀

**Review by Domiii — COMMENTED** [review]
I'm not sure I understand this 😅
Why are the Mac steps gone?

**Comment by toshok** [comment]
> I'm not sure I understand this 😅 Why are the Mac steps gone? (I am guessing this is work in progress?)

ah yeah I should have flipped it back to a draft. I was still iterating and didn't want to wait for those steps so the buildkite pipeline got trimmed to basically nothing.  work is done now :)

**Review by Domiii — APPROVED** [review]

**Pull request merged**